kolibri/core/tasks/compat.py

```diff
@@ -1,19 +1,28 @@
+from kolibri.utils.conf import OPTIONS
+
 try:
+    if not OPTIONS["Tasks"]["WORKER_MULTIPROCESSING"]:
+        raise ImportError()
     # Import in order to check if multiprocessing is supported on this platform
-    from multiprocessing import synchronization  # noqa
+    from multiprocessing import synchronize  # noqa
 
     # Proxy Process to Thread to allow seamless substitution
     from multiprocessing import Process as Thread  # noqa
     from multiprocessing import Event  # noqa
 
     # any variable is local to a process, so this is
     # just a dummy
-    local = object
+    class local(object):
```

Comment:
Interesting, what's the reason for the additional class definition here?

Reply:
I think you can't actually directly instantiate `object` because it's an ABC.